### DISCUSSION TITLE
Adding Numbat to Nixpkgs

### DISCUSSION THREAD

**Discussion by giomf** [sharkdp/numbat#215] [Ideas]
Hello,

I wanted to check if you are interested in me adding numbat to nixpkgs so nixos user can install numbat easier?
It should be mentioned that I am relatively new to nixos and I might need some time.

**Comment by sharkdp** [comment]
That would be great - yes!

**Comment by giomf** [comment]
Ok.
I will start right away!

**Comment by giomf** [comment]
I already created a pull request: https://github.com/NixOS/nixpkgs/pull/264241 and it looks quite good.
Hope it will get merged soon.

**Comment by giomf** [comment]
PR has been merged into master.
We have to wait for all backports and then its done.
https://nixpk.gs/pr-tracker.html?pr=264241

I will open a PR to adapt the docs how to install numbat via nix.

**Reply by sharkdp** [reply]
Thank you. Have you seen https://numbat.dev/doc/cli-installation.html#guidelines-for-package-maintainers? It's not necessary — but if you want to provide the best possible experience for NixOS users, you could follow the instructions there.

> I will open a PR to adapt the docs how to install numbat via nix.

:+1:

**Reply by giomf** [reply]
Numbat just landet into all backports 🎉 

> Have you seen https://numbat.dev/doc/cli-installation.html#guidelines-for-package-maintainers?

Yes, but I didnt figure out how to do it in a nix way yet. But it will be my next tasks!

**Reply by giomf** [reply]
This will add the prelude to the package:
https://github.com/NixOS/nixpkgs/pull/265744

**Comment by triallax** [comment]
> I wanted to check if you are interested in me adding numbat to nixpkgs so nixos user can install numbat easier?

For the record, I just want to clarify that nobody needs permission to package Numbat for any repository, so if you would like to give it a stab, then go right ahead! Of course, we would still love to know if you did, or are working to, package Numbat, and if you face any issues we are here to help, so ask away. :)
